.github/actions/merge-oci-digests/action.yaml

```diff
@@ -16,7 +16,7 @@ runs:
   using: composite
   steps:
   - name: Retrieve OCI digests
-    uses: actions/download-artifact@v4
+    uses: actions/download-artifact@v3
```

Comment:
still we should use v4 everywhere and adapt if necessary

Reply:
only this comment from my side, we can merge after

Reply:
I will wait for https://github.com/unikraft/catalog/pull/156 then rebase